Woof/Woof/Model/Sitter/Sitter+DummyData.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+extension Sitter {
+    static let dummySitter = Sitter(
+        id: UUID(),
```

Comment:
How we're going to use the dummy sitter?
Should it have the same `id` each time we run the app?

Reply:
We will use this dummy sitter for previews only, add the doc comment.
Fixed  with https://github.com/ios-course/ironfoudation-team-project/pull/38/commits/e3cecfe033d1f4b3263e510ebfde6c655f755d29